api/resources_portal/test/test_data_readme.txt

```diff
@@ -0,0 +1,45 @@
+The test data describes the following situation:
```

Comment:
This file is great, but let's make it markdown instead of a `.txt`. I think we can just make this a new section of the documentation. https://github.com/AlexsLemonade/resources-portal/tree/dev/api/docs/api will be good place for it, because then once we get https://github.com/AlexsLemonade/resources-portal/issues/45 it'll be incorporated nicely.

In that same doc you can also mention how to load it up once you get that sorted.